### PR TITLE
Fix app dev documentation

### PR DESCRIPTION
**Pull request by fabolhak**
This solves some rendering problems for tables [here](https://boschrexroth.github.io/ctrlx-automation-sdk/latest/appdevguide_basechecks.html)

### CODE REVIEW

File: doc/appdevguide_basechecks.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The content of this document may be subject of change in further versions of the
 ## 1.2 Additional notes
 
 - **Schema for snapcraft.yaml:** <https://github.com/snapcore/snapcraft/blob/master/schema/snapcraft.json>
-- **Schema for technical app  name:** "ctrlx-{company name}-{app name}-{app version}_arm64/amd64.snap"
+- **Schema for technical app name:** "ctrlx-{company name}-{app name}-{app version}_arm64/arm64.snap"
```
[the remark this answers is not in the document]
Comment (guidfort) [reply doc/appdevguide_basechecks.md:30]: No it is not a typo. It means that the ending is either "arm64" or "amd64" (arm64/amd64 == arm64 'or' amd64)

[the remark this answers is not in the document]
Comment (guidfort) [reply doc/appdevguide_basechecks.md:30]: Here it is wrong, in the original docu. it is correct.

**Comment by guidfort** [comment]
Original documentation has been updated, so PR will be closed.

File: doc/appdevguide_basechecks.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fabolhak) [reply doc/appdevguide_basechecks.md:30]: oh sorry. That was my fault. I interpreted the forward slash as folder separator (which does not make too much sense in hindsight). Thank you for fixing this

**Pull request closed without merging**